core/modules/mem_mdlw/src/in_memory_queue_poller.cpp

```diff
@@ -30,12 +30,16 @@ namespace mem
 
     void InMemoryQueuePoller::stop() {
         _running = false;
-        if(_threadNotifier.joinable()) _threadNotifier.join();
+        if(_threadNotifier.joinable()) {
+		    _threadNotifier.join();
+        }
     }
 
     InMemoryQueuePoller::~InMemoryQueuePoller() {
         _running = false;
-        if(_threadNotifier.joinable()) _threadNotifier.join();
+        if(_threadNotifier.joinable()) {
+		    _threadNotifier.join();
```

Comment:
Indent should be 4 spaces to match rest of code.

Reply:
done